### ISSUE TITLE
Event text colour improvement

### ISSUE THREAD

**Issue by OddHouseGames** [ProtonMail/proton-calendar#8]
iOS Safari

Currently the event text is black on dark grey which is not great for accessibility.

Improvements could include either lightening the colour of the event box, or darkening it and making the text a lighter colour.

This would benefit regular users and those with any accessibility issues that require more contrast when reading digit text.![image](https://user-images.githubusercontent.com/62911518/111900857-3564bd80-8a0b-11eb-8cc2-a07dc5eecc5b.jpeg)

**Comment by econdepe** [comment]
Better contrast will be added soon, probably for ProtonCalendar beta 17.
